scripts/SANS/sans/gui_logic/presenter/add_runs_presenter.py

```diff
@@ -134,13 +123,15 @@ def _handle_selection_changed(self, run_selection):
     def _handle_out_file_changed(self):
         self._use_generated_file_name = False
 
-    def _output_directory_is_not_empty(self, settings):
+    @staticmethod
+    def _output_directory_is_not_empty(settings):
         return settings.save_directory != ''
 
     def _handle_output_directory_changed(self):
         directory = self._view.display_save_directory_box("Save sum runs", self.save_directory)
-        self._update_output_directory(directory)
-        self._view.set_out_file_directory(self.save_directory)
+        directory = os.path.join(directory, '')  # Add an OS specific trailing slash if it doesn't already exist
```

Comment:
As a more explicit alternative you could use `directory + os.sep`, but the `.join` with the comment is just as good

Reply:
using join will only add a slash if it doesn't end in one. I had noticed that sometimes when you selected an output directory it would end in a slash and other times it wouldn't. 

I didn't know about os.sep, though, so I'll keep that in mind for cases where we always need to add one